modules/deploy/pages/deployment-option/cloud/remote-read-replicas.adoc

```diff
@@ -0,0 +1,57 @@
+= Remote Read Replicas
+:description: Learn how to create a remote read replica topic, which is a read-only topic that mirrors a topic on a different cluster.
+:page-cloud: true
+
+A remote read replica topic is a read-only topic that mirrors a topic on a different cluster. You can create a separate remote cluster just for consumers of this topic and populate its topics from object storage. A read-only topic on a remote cluster can serve any consumer, without increasing the load on the source cluster. Because these read-only topics access data directly from object storage, there's no impact to the performance of the cluster.
+
+Redpanda Cloud supports remote read replicas with ephemeral BYOC clusters (not with a customer-managed VPC). Ephemeral clusters are temporary clusters created to handle specific, transient workloads or tasks. After processing, the cluster is terminated to free up resources and reduce costs. 
+
+== Prerequisites
+
+* A BYOC source cluster in Ready state.
+* A BYOC reader cluster in Ready state. This separate reader cluster must exist in the same Redpanda organization and the same cloud provider account and region as the source cluster.
+
+== Configure remote read replicas
+
+You can create remote read replica topics in Redpanda Cloud with the xref:api:ROOT:cloud-api.adoc[Cloud API]. For information on accessing the Cloud API, see xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[].
+
+. To update your source cluster to add one or more reader cluster IDs, use the PATCH operation. The full list of clusters is expected on every call. If an ID is removed from the list, it will be removed as a reader cluster.
++
+```bash
+export SOURCE_CLUSTER_ID=.......
+export READER_CLUSTER_ID=.......
+
+curl -X PATCH $API_HOST/v1beta1/clusters/$SOURCE_CLUSTER_ID \
```

Comment:
We recently launched `v1beta2` so we should probably use that for any new documentation. In this case it's exactly the same, just change to `v1beta2`